t/005-exit.t

```diff
@@ -488,6 +488,22 @@ GET /lua
 
 
 === TEST 13: 501 Method Not Implemented
+--- config
+    location /lua {
+        content_by_lua '
+            ngx.exit(ngx.HTTP_METHOD_NOT_IMPLEMENTED)
+        ';
+    }
+--- request
+GET /lua
+--- error_code: 501
+--- response_body_like: 501 (?:Method )?Not Implemented
+--- no_error_log
+[error]
```

Comment:
It's better to add new test blocks at the end of file, to avoid affecting the test titles in subsequent test blocks, leading to higher chance of merge conflicts.

Also, we should update the `doc/HttpLuaModule.wiki` file and `README.markdown` for this new constant.